packages/image_picker/image_picker_ios/lib/image_picker_ios.dart

```diff
@@ -51,14 +51,28 @@ class ImagePickerIOS extends ImagePickerPlatform {
   }) async {
     final String? path = await _pickImageAsPath(
       source: source,
-      maxWidth: maxWidth,
-      maxHeight: maxHeight,
-      imageQuality: imageQuality,
-      preferredCameraDevice: preferredCameraDevice,
+      options: ImagePickerOptions(
+        maxWidth: maxWidth,
+        maxHeight: maxHeight,
+        imageQuality: imageQuality,
+        preferredCameraDevice: preferredCameraDevice,
+      ),
     );
     return path != null ? PickedFile(path) : null;
   }
 
+  @override
+  Future<XFile?> getImageFromSource({
```

Comment:
Maybe we should deprecate `pickImage` as `getImageFromSource` is the new method we want users to start using instead.

Reply:
Actually deprecating platform interface methods is a multi-step process; see discussion of deprecation in [this section of the docs](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters)